app/models/member.rb

```diff
@@ -15,6 +15,10 @@ class Member < ApplicationRecord
   before_create :set_joined_at
   after_create :reset_token!
 
+  def confirm!
```

Comment:
Yes, I like that! I was thinking that it would be good if we also confirm their active-ness when they register to vote.

Reply:
> Yes, I like that! I was thinking that it would be good if we also confirm their active-ness when they register to vote.

Yes! That is a great idea :tada:.